IdokladSdk/Validation/Attributes/DateGreaterThanOrEqualThanAnotherDateAttribute.cs

```diff
@@ -17,23 +19,33 @@ public DateGreaterThanOrEqualThanAnotherDateAttribute(string otherPropertyName)
 
         protected override ValidationResult IsValid(object value, ValidationContext validationContext)
         {
+            var propertyInfo = validationContext.ObjectType.GetProperty(validationContext.MemberName);
             var otherPropertyInfo = validationContext.ObjectType.GetProperty(OtherPropertyName);
             var otherPropertyType = GetPropertyType(otherPropertyInfo);
 
-            if (otherPropertyType == default(DateTime).GetType())
+            if (propertyInfo != null)
             {
-                if (value == null)
+                if (value == null || (_allowDefualtDate && (DateTime)value == Constants.DefaultDateTime))
```

Comment:
co když je to typu DateTime?

Reply:
Tak se to přetypuje na DateTime. Null to určitě nebude, protože pokud by to bylo null, tak se k té podmínce za logickým or || ani kompilátor nedostane, vyhodnotí podmínku value == null a jde rovnou do těla ifu.
Nebo jsi viděl ještě nějaký jiný problém?

Reply:
trapas. Je to správně

Reply:
_allowDefualtDate typo chyba v Defualt